src/test/groovy/graphql/nadel/NadelE2ETest.groovy

```diff
@@ -680,4 +680,84 @@ class NadelE2ETest extends Specification {
 
     }
 
+    def "test hydration inside a renamed field"() {
```

Comment:
I think this test should be in NadelExecutionStrategyTest and be more specific: see the other tests there